core/embed/prodtest/README.md

```diff
@@ -106,6 +106,48 @@ TOUCH 09
 OK 50 90
 ```
 
+
+### TOUCH_CUSTOM
+The `TOUCH_CUSTOM` command tests the functionality of the display's touch screen.
+It draws a filled rectangle on custom coordinates and waits for user interaction.
+
+The command requires five input parameters:
+
+* X position of the top-left corner of the rectangle
+* Y position of the top-left corner of the rectangle
+* Width of the rectangle
+* Height of the rectangle
+* The timeout in seconds
+
+If the display is not touched within the specified timeout, the command will return an `ERROR TIMEOUT`.
+
+The device report touch events, coordinates and timestamps (in ms), correctness of the touch point is not checked and is left to the test equipment.
+
+The test ends with first lift-up event.
+
+Example (to draw a 100x100 rectangle in the top-left (10,10) position and wait for 15 seconds for touch input):
+```
+TOUCH_CUSTOM 10 10 100 100 15
+TOUCH D 69 35 357300
+TOUCH U 69 35 357328
+OK
+```
+
+### TOUCH_IDLE
+The `TOUCH_IDLE` command tests the functionality of the display's touch screen.
+It waits for a specific time period without any touch input.
+
+The command requires one input parameter:
+* The timeout in seconds
+
+If a touch event is detected within the specified timeout, the command will return an `ERROR TIMEOUT`, the touch event, its coordinates and timestamp (in ms) are reported.
```

Comment:
The behavior is different from what is described here.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/aed6c9e3ff877a30cc011fd248840cd056e11cd5